apps/test/unit/weblab/WebLabTest.js

```diff
@@ -124,4 +201,80 @@ describe('WebLab', () => {
       });
     });
   });
+
+  describe('getCodeAsync', () => {
+    it('resolves with empty string if brambleHost is null', () => {
+      weblab.brambleHost = null;
+      weblab.getCodeAsync().then(value => {
+        expect(value).to.equal('');
+      });
+    });
+    it('rejects with error if brambleHost syncFiles has an error', () => {
+      weblab.brambleHost = {
+        syncFiles: callback => {
+          callback('error');
+        }
```

Comment:
nit: this expression can be simplified to `syncFiles: callback => callback('error')`

Reply:
(same on lines 224-226 below)